### PR TITLE
Mip tests and tuntap example to compile on OpenBSD natively.

### PR DESCRIPTION
**Pull request by jfsimon1981**
Tuntap related calls modified to compile natively on OpenBSD.

### CODE REVIEW

File: examples/mip-tap/main.c [hunk 1]
```diff
@@ -78,6 +99,10 @@ int main(int argc, char *argv[]) {
   mip_init(&mgr, &mif);
   MG_INFO(("Init done, starting main loop"));
 
+  // Start infinite event loop
+  MG_INFO(("Mongoose version : v%s", MG_VERSION));
+  MG_INFO(("Listening on     : %s", s_listening_address));
+
   extern void device_dashboard_fn(struct mg_connection *, int, void *, void *);
   mg_http_listen(&mgr, "http://0.0.0.0:8000", device_dashboard_fn, &mgr);
```
Comment (scaprile) [inline examples/mip-tap/main.c:107]: `mg_http_listen(&mgr, s_listening_address, device_dashboard_fn, &mgr);`

**Review by scaprile — COMMENTED** [review]
Minor detail

File: examples/mip-tap/main.c (hunk 1 above)
Comment (jfsimon1981) [reply examples/mip-tap/main.c:107]: Failed ;-( thank you

Comment (scaprile) [reply examples/mip-tap/main.c]: Just "humans being"

**Pull request merged**